bors/src/command.rs

```diff
@@ -13,7 +13,7 @@ use thiserror::Error;
 
 #[derive(Error, Debug)]
 #[error("invalid command")]
-pub struct ParseCommnadError;
+pub struct ParseCommandError;
```

Comment:
🤦  What's worse is that the misspelled type name is used significantly more than once. Thanks for fixing this

Reply:
I'm glad my OCD can come in handy ;)